analysis_options.yaml

```diff
@@ -18,6 +18,19 @@ analyzer:
   exclude: # DIFFERENT FROM FLUTTER/FLUTTER
     # Fixture depends on dart:ui and raises false positives.
     - flutter_frontend_server/test/fixtures/lib/main.dart
+    # FIXME: DO NOT SUBMIT.
+    # We have a root `.dart_tool/packages_config.json`, so the entire repo is
+    # automatically treated as a single Dart package unless a pubspec.yaml file
+    # exists. Some of these directories do not follow that pattern and need to
+    # be ignored.
+    - lib/**
```

Comment:
It isn't clear to me why these need to be ignored?

Reply:
Wasn't for me either. I think having the root package config for the first time starts considering everything as one big package, which means there are some analysis issues that were hidden before.

I'll share some logs of what it looks like.